iosApp/iosApp/app/NotificationsApiImpl.swift

```diff
@@ -21,7 +23,52 @@ class NotificationsApiImpl : NSObject, NotificationsApi {
     
     let notificationDelegate = LocalNotificationDelegate()
     
-    func createLocalNotification(title: String, message: String, timeInMS: Int64, notificationId: Int32) {
+    
+    func createReminderNotificationsForSessions(sessions: [MySessions]) {
+        if(NotificationsKt.reminderNotificationsEnabled() && NotificationsKt.notificationsEnabled()) {
+            for session in sessions {
+                let notificationTime = session.startsAt.toLongMillis() - Int64(AppContext().TEN_MINS_MILLIS)
+                if (notificationTime > FunctionsKt.currentTimeMillis()) {
+                    createLocalNotification(title: "Upcoming Event in " + session.roomName,
+                                            message: session.title + " is starting soon.",
+                                            timeInMS: notificationTime,
+                                            notificationId: Int32(session.id)!,
+                                            notificationTag: NotificationsApiKt.notificationReminderTag)
+                }
+            }
+        }
+    }
+    
+    func createFeedbackNotificationsForSessions(sessions: [MySessions]) {
+        if(NotificationsKt.feedbackEnabled() && NotificationsKt.notificationsEnabled()) {
+            for session in sessions {
```

Comment:
Will this make a notification for every rsvp'd session?

Reply:
Yea, my mistake. I've updated this in a new commit.